packages/google_maps_flutter/google_maps_flutter_ios/example/ios11/ios/Podfile

```diff
@@ -1,5 +1,5 @@
-# Uncomment this line to define a global platform for your project
-# platform :ios, '11.0'
+# Global platform version is set to 12 for this example project to support cloud-based maps styling
```

Comment:
The iOS 11 example should not be changed to use iOS 12. That's what the iOS 12 and iOS 13 examples are for.

Reply:
Of course not, accidentally missed this in rebase. Change reverted.